src/function/lazyLoading.js

```diff
@@ -7,13 +7,20 @@ function($) {
         if (!self.sprite && self.lasyEmoji[0]) {
             var pickerTop = self.picker.offset().top,
                 pickerBottom = pickerTop + self.picker.height() + 20;
-            self.lasyEmoji.each(function() {
+
+            self.picker.find(".lazy-emoji").each(function() {
                 var e = $(this), top = e.offset().top;
+
                 if (top > pickerTop && top < pickerBottom) {
                     e.attr("src", e.data("src")).removeClass("lazy-emoji");
                 }
-            })
+
+                // improve performance by early-exit when we encounter first emoji below bottom of picker viewport
+                if (top > pickerBottom) {
```

Comment:
This will not work, because it can loads in the both side from top to bottom as same as from bottom to top, when filter button was clicked before lasyLoad

Reply:
yes, you are right